File: 프로그래머스/0/120812. 최빈값 구하기/최빈값 구하기.py
def solution(array):
    answerArr = [0] * (max(array)+1)
    for v in array:
        answerArr[v] += 1
    
    if answerArr.count(max(answerArr)) != 1: return -1 
    else: return answerArr.index(max(answerArr))

File: 프로그래머스/0/120812. 최빈값 구하기/최빈값 구하기.js
function solution(array) {
    let answerArr = new Array(Math.max(...array)+1).fill(0);
    
    for (let i = 0; i < array.length; i++) {
        answerArr[array[i]] += 1;
    }
    
    let mode = Math.max(...answerArr)
    
    if (answerArr.indexOf(mode) !== answerArr.lastIndexOf(mode)) {
        return -1
    } else {
        return answerArr.indexOf(mode);
    };
}